modules/geopackage/src/lib/parse-geopackage.js

```diff
@@ -19,6 +31,26 @@ const ENVELOPE_BYTE_LENGTHS = {
   7: 0
 };
 
+// Documentation: https://www.geopackage.org/spec130/index.html#table_column_data_types
+const SQL_TYPES = {
+  BOOLEAN: Bool,
+  TINYINT: Int8,
+  SMALLINT: Int16,
+  MEDIUMINT: Int32,
+  INT: Int32,
+  INTEGER: Int32,
+  FLOAT: Float32,
+  DOUBLE: Float64,
+  REAL: Float64,
+  TEXT: Utf8,
+  BLOB: Binary,
+  DATE: Utf8,
+  DATETIME: Utf8,
+  GEOMETRY: Binary
```

Comment:
While it's true that a geometry is internally stored as binary (WKB), that's an incorrect output schema type, since this loader converts it to GeoJSON.